1.8/administration/banning

```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title:  "Banning"
+categories: [administration]
+---
+
+The MyBB Admin CP Banning page, found under the "Configuration" tab, offers the ability to ban IP Addresses and disallow usernames and email addresses. Wildcard support is included for IP addresses, usernames, and email addresses, and for all you can view the date banned and the last attempted use. More information for each sub-page is found below.
+
+## Banned IPs
+
+Banned IP addresses disallow access for anyone accessing your MyBB forum from the IP addresses banned. Also supported is banning ranges of IP addresses, using \* to represent the wildcard. You can view when each IP address was banned, and when the IP address last attempted to access the forum.
+
+#### Banning IPs
+
+To ban an IP address, just enter the IP address, or range of IP addresses using an asterisk (\*) to represent the wildcard, into the form at the bottom of the page and select "Ban IP Address."
```

Comment:
This should probably read as:
To ban an IP address, just enter the IP address, or range of IP addresses using an asterisk (\*) to represent the wildcard, into the form and select "Ban IP Address."